src/NHSD.BuyingCatalogue.Identity.Api/Settings/DisabledErrorMessageSetting.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace NHSD.BuyingCatalogue.Identity.Api.Settings
+{
+    public sealed class DisabledErrorMessageSetting
```

Comment:
Due to the multiple properties, the class name to me suggests a plural nature e.g. `DisabledErrorMessageSettings`. This is just a suggestion, don't have to adhere.